encfs/encfssh

```diff
@@ -6,16 +6,17 @@
 # Contributed by David Rosenstrauch.
```

Comment:
My line of thinking was to do everything before the mount; But yes, probably fine to keep it like this. But probably should get some more error handling.